test/e2e/serverside_apply_test.go

```diff
@@ -47,11 +47,11 @@ func serversideApplyTest(c client.Client, invConfig InventoryConfig, inventoryNa
 	}, &d)
 	Expect(err).NotTo(HaveOccurred())
 	_, found := d.ObjectMeta.Annotations[v1.LastAppliedConfigAnnotation]
-	Expect(found).To(Equal(false))
+	Expect(found).To(BeFalse())
```

Comment:
We didn't add sufficient comments initially, but we might want to comment somewhere about the difference of client-side (sets `LastAppliedConfigAnnotation`) vs. server-side apply (sets field managers) to help explain the tests.